en/sim_gazebo_gz/README.md

```diff
@@ -218,6 +261,11 @@ To add a new model:
      PX4_SYS_AUTOSTART=<your new airframe id> ./build/px4_sitl_default/bin/px4
      ```
 
+To add a new world:
+
+1. Add your world to the list of worlds found [here](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/simulation/gz_bridge/worlds_list).
+   This is required in order to allow CMake to generate correct targets.
+
 :::note
 As long as the world file and the model file are in the Gazebo search path `GZ_SIM_RESOURCE_PATH` it is not necessary to add them to the PX4 world and model directories.
```

Comment:
So can you access worlds/vehicles still without adding them to the repo?

Reply:
For worlds, you only need to add the name of your world to the `worlds_list` file. For vehicles, you only need to add the airframe file to PX4 (and add that file to CMake) and then place the respective sdf files in a place where they lie on the GZ_SIM_RESOURCE_PATH. Per default this is in `/.simulation-gazebo` but you can modify this.

Reply:
I think this is worth saying (right)? Can you adjust the docs at this point if you think they need any work in this respect?

Reply:
This is outdated now that the worlds list no longer exist. I've reworked the above mentioned section to explain the fact that we are now using a submodule and a different CMake structure.